Core/Core/Login/LoginStartViewController.storyboard

```diff
@@ -339,7 +339,7 @@
                                     <constraint firstItem="Vec-s2-2D4" firstAttribute="leading" secondItem="xBc-5O-QU6" secondAttribute="leading" id="GCc-Dr-KkO"/>
                                     <constraint firstAttribute="trailing" secondItem="k72-wA-dRu" secondAttribute="trailing" constant="16" id="NxQ-js-qpq"/>
                                     <constraint firstItem="Op3-kc-wqp" firstAttribute="leading" secondItem="xBc-5O-QU6" secondAttribute="leading" constant="16" id="QqB-yW-Rd2"/>
-                                    <constraint firstAttribute="height" priority="750" constant="225" id="WS4-Gy-0Kt"/>
+                                    <constraint firstAttribute="height" priority="500" constant="225" id="WS4-Gy-0Kt"/>
```

Comment:
This was the core fix. Break the height of the previous logins before breaking the height of the login button.